Utilities/GenerateDocs/docker/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:18.04
+
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y cmake cmake-curses-gui gcc g++ git python python-dev doxygen ninja-build gnuplot wget texlive-full
```

Comment:
Does this version of doxygen work or does it generate the `arrows` that do not expand? Otherwise, you could also install doxygen directly from scratch like I have done it [here](https://github.com/InsightSoftwareConsortium/ITKDevelopmentEnvironment/blob/master/Docker/simpleitk-doxygen/Dockerfile)

Reply:
Here is a cononical link to your fix:
https://github.com/InsightSoftwareConsortium/ITKDevelopmentEnvironment/blob/116ebd57d5ea5cde8618a16be51ffda8905120f0/Docker/simpleitk-doxygen/Dockerfile#L24-L28

This version is unfortunately also affected by the bug.